com.amazon.Workspaces.yml

```diff
@@ -165,12 +146,6 @@ modules:
       - type: extra-data
         filename: workspacesclient.deb
         only-arches: [x86_64]
-        url: https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu/dists/focal/main/binary-amd64/workspacesclient_4.7.0.4312_amd64.deb
-        sha256: 1b4a39b859c49225169a44cc507955722770fb6c7c7bf2a631f54113ba0b5d5f
-        size: 46652448
-        x-checker-data:
-          type: debian-repo
-          package-name: workspacesclient
-          root: https://d3nt0h4h6pmmc4.cloudfront.net/ubuntu
-          dist: focal
-          component: main
+        url: https://d3nt0h4h6pmmc4.cloudfront.net/new_workspacesclient_jammy_amd64.deb
```

Comment:
If you pin this to a specific version, the SHA will be predictable and won't break when future builds are released.

Reply:
That's relevant if I can get it to work. The last status was it didn't work.

Reply:
Good luck! I don't know enough to fix it, but I'm rooting for you. Having this available as a Flatpak would be great for me, as I need to remote into a work VDI on WorkSpaces frequently :)

Reply:
Rooting for this as well @bbhtt. It would be nice to have this as an option. Sadly we'll probably need the PCoIP version as well.

Reply:
I think for pcoip you would use an older version that supports it